spec/steps/claim_details/form_spec.rb

```diff
@@ -62,47 +74,77 @@
 
       context 'time_spent has a value in the database' do
         let(:time_spent_in_db) { 100 }
+        let(:work_before_date) { Date.new(2023, 1, 1) }
+        let(:work_after_date) { Date.new(2023, 1, 1) }
```

Comment:
these additional lets are not required here as they are already added at the top level